### DISCUSSION TITLE
Is there a way to use Quarto on the Planetary Computer Hub?

### DISCUSSION THREAD

**Discussion by cullen-molitor** [microsoft/PlanetaryComputer#60] [Ideas]
I think [Quarto](https://quarto.org/) could be a nice addition for teams developing notebooks collaboratively. The plain text aspect makes handling conflicts far simpler than Jupyter Notebooks.

**Comment by TomAugspurger** [comment]
I haven't used Quarto before. Do you know if / how it integrates with JupyterHub?

**Reply by cullen-molitor** [reply]
I am unsure about configuring JupyterHub, but it is compatible with [JupyterLab](https://quarto.org/docs/tools/jupyter-lab.html). I am mostly interested in the [plain text editing](https://quarto.org/docs/tools/jupyter-lab.html#plain-text-editing) feature but it looks like it could be a bit clumsy to run a `.qmd` file in JupyterLab. 

There is a VS Code extension that looks like a nicer way to edit and run chunks. I think this might be the way to go for using Quarto.
